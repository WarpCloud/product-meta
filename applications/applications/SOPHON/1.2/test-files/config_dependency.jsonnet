{
  application_type: "SOPHON",
  application_name: "sophon1",
  application_version: "1.2",
  user_config: {},
}
