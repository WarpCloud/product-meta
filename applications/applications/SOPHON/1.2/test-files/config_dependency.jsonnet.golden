{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "sophon1",
            "application_type": "SOPHON",
            "use_hyperbase": "true",
            "use_inceptor": "true",
            "use_search": "true"
         },
         "configs": { },
         "instance_settings": { },
         "moduleName": "redis",
         "name": "sophon1-redis",
         "version": "1.0"
      },
      {
         "annotations": {
            "application_name": "sophon1",
            "application_type": "SOPHON",
            "use_hyperbase": "true",
            "use_inceptor": "true",
            "use_search": "true"
         },
         "configs": {
            "sophon_cpu_limit": 2,
            "sophon_cpu_request": 0.10000000000000001,
            "sophon_memory_limit": 8,
            "sophon_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "hdfs",
               "name": "hdfs"
            },
            {
               "moduleName": "yarn",
               "name": "yarn"
            },
            {
               "moduleName": "zookeeper",
               "name": "zookeeper"
            },
            {
               "moduleName": "txsql",
               "name": "txsql"
            },
            {
               "moduleName": "redis",
               "name": "sophon1-redis"
            },
            {
               "moduleName": "hyperbase",
               "name": "hyperbase"
            },
            {
               "moduleName": "inceptor",
               "name": "inceptor"
            },
            {
               "moduleName": "elasticsearch",
               "name": "elasticsearch"
            }
         ],
         "instance_settings": { },
         "moduleName": "sophonweb",
         "name": "sophon1-sophonweb",
         "version": "1.2"
      }
   ],
   "instance_settings": { },
   "version": ""
}
